lxd/instance/drivers/driver_qemu.go

```diff
@@ -4964,3 +4964,47 @@ func (d *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Info returns "qemu" and the currently loaded qemu version.
+func (d *qemu) Info() instance.Info {
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
+	data := instance.Info{
+		Name:    "",
```

Comment:
Name should always be `qemu`, that's the name of the driver.

Reply:
Fixed in 37c1ac5